### ISSUE TITLE
Portus image not updating.

### ISSUE THREAD

**Issue by thomas15v** [openSUSE/docker-containers#107]
https://hub.docker.com/r/opensuse/portus/builds/

Looks like docker hub is not happy.

**Comment by flavio** [comment]
@mssola can you take a look into that?

**Comment by mssola** [comment]
@flavio yes, it appears that the gpg trick that we used to do is not working anymore for an unknown reason... I tried to fetch the key from other providers, and all this works locally, but on Docker hub it either times out, or refuses to connect.

I guess I'll take a further look into the gpg command, to see if there's something I can use. Otherwise, I'm afraid that the only option left is to blindly accept the gpg key from V:c:Portus and be done with it :thinking:

**Comment by flavio** [comment]
> I guess I'll take a further look into the gpg command, to see if there's something I can use. Otherwise, I'm afraid that the only option left is to blindly accept the gpg key from V:c:Portus and be done with it thinking

I hope that's not going to be the case!

**Comment by mssola** [comment]
I already fixed the issue for the `2.4` and `latest` images. Now only `head` is missing, which looks like a regression of something me and @jordimassaguerpla already fixed some weeks ago...
